roles/remediate/tasks/leapp_pam_tally2.yml

```
@@ -0,0 +1,20 @@
+---
+- name: leapp_pam_tally2 | Remove pam_tally2 modules
+  # pam_tally2 only existed in RHEL 5-7, skip if this is not a RHEL 7 upgrade
+  when: ansible_distribution == 'RedHat' and ansible_distribution_major_version|int == 7
+  block:
+    - name: leapp_pam_tally2 | Find all instances of pam_tally2 in /etc/pam.d/
+      ansible.builtin.find:
+        paths: /etc/pam.d/
+        file_type: file
+        contains: "pam_tally2.so"
+        read_whole_file: true
+        recurse: true
+      register: file_list
+
+    - name: leapp_pam_tally2 | Comment out all instances of pam_tally2 # noqa: command-instead-of-module
+      ansible.builtin.command:
+        cmd: "sed '/^#/! {/pam_tally2/ s/^/#/}' < {{ item }}"
```

Looks to me like this is just going to write the changes to stdout, but not make changes to the files. Maybe this instead?... 
```suggestion
        cmd: "sed -i '/^#/! {/pam_tally2/ s/^/#/}' {{ item }}"
```

---

Thanks for the catch, was trying to write it in a way that would work with ansible.builtin.replace and wasn't getting it to cooperate with the two commands (so I don't add additional comments when it reruns)

Working on refactoring out the end_hosts now